src/super_gradients/training/metrics/segmentation_metrics.py

```diff
@@ -122,6 +122,18 @@ def intersection_and_union(im_pred, im_lab, num_class):
     return area_inter, area_union
 
 
+def _map_ignored_inds(target: torch.Tensor, ignore_index_list, unfiltered_num_classes) -> torch.Tensor:
```

Comment:
I feel this method needs docs, especially what is unfiltered_num_classes.
Maybe think of a better name for this variable? `num_evaluated_classes` maybe?

Reply:
Added c681c262f3e962b1faa3317e7fca7dd7d7bdbb37